media-tv/mythtv/mythtv-30.0.ebuild

```diff
@@ -0,0 +1,408 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+
+# Specify a specific commit to download.  If we just pull the latest from the branch, the ebuild
+# digest will break.  Update this periodically.  When updating, keep old versions so that the
+# ebuild will work on all of them with simple renaming.
+BACKPORTS="5cde0578d84926171b20c8f7e95a101e9b0b9457" # August 8, 2019
+
+MY_P=${P%_p*}
+MY_PV=${PV%_p*}
+
+PATCHES=(
+	# Selected a fixes/... branch commmit to get rid of previous patches
+)
+
+inherit eutils flag-o-matic python-single-r1 qmake-utils user readme.gentoo-r1 systemd vcs-snapshot
+
+MYTHTV_BRANCH="fixes/${P%.*}"
+
+DESCRIPTION="Homebrew PVR project"
+HOMEPAGE="https://www.mythtv.org"
+SRC_URI="https://github.com/MythTV/mythtv/archive/${BACKPORTS}.tar.gz -> ${PF}.tar.gz"
+
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~x86"
+SLOT="0/${PV}"
+
+IUSE_INPUT_DEVICES="input_devices_joystick"
+# Fixme: These should be video_capture_... flags to keep them separate, but I'm not sure
+# creating a new flag category is appropriate for just this ebuild, and several of the flags
+# already exist in other ebuilds
+IUSE_VIDEO_CAPTURE_DEVICES="v4l ivtv ieee1394 hdpvr hdhomerun vbox ceton"
+IUSE="alsa altivec autostart bluray cdda cdr cec debug dvd egl exif fftw jack java lcd libass
+	lirc perl pulseaudio python raw systemd vaapi vdpau vpx +wrapper x264 x265 +xml xmltv +xvid
+	zeroconf ${IUSE_INPUT_DEVICES} ${IUSE_VIDEO_CAPTURE_DEVICES}"
+# Notes on use flags:
```

Comment:
I think having "client" & "server" like USEs here isnt bad idea? Can you make a prototype without updating ebuild how you'd write it?
Anyway these thoughts should be dropped from this ebuild. You can open a bug about it if you wish.